### DISCUSSION TITLE
Stitches Mixins

### DISCUSSION THREAD

**Discussion by Code-Victor** [ora-ui/ora-ui#66] [Ideas]
I found this awesome [library](https://github.com/joe-bell/stitches-mixins) by [Joe Bell](https://github.com/joe-bell) that allows the use of mixins in stitches.
```jsx
// with stitches-mixins
const Label = styled("label", {
  include: "screenReaderOnly",
  // *or* include multiple…
  include: ["screenReaderOnly", "minHeightScreen"],
});
```
It comes with some nice defaults but you can add some more in the stitches config.
```jsx
export const { css, styled } = createStitches({
  theme: {},
  utils: {
    // with custom mixins
    include: mixins({
      orchidShadow: {
        boxShadow: "0 25px 50px -12px orchid",
      },
    }),
    // …or without
    include: mixins(),
  },
});
```

**Comment by gregogun** [comment]
This is very cool, I wasn't aware of this! 

This would be very useful indeed. Would you be happy to set up a pr for this?

**Comment by Code-Victor** [comment]
Alright, I'll do just that
